custom-config.fcc

```diff
@@ -53,9 +44,8 @@ storage:
           fi
           export PRIMARY_MAC=$(echo $macs | awk -F, '{print $1}')
           export SECONDARY_MAC=$(echo $macs | awk -F, '{print $2}')
-          mount "$device"1 /var/mnt
-          echo -e "PRIMARY_MAC=${PRIMARY_MAC}\nSECONDARY_MAC=${SECONDARY_MAC}" > /var/mnt/mac_addresses
-          umount /var/mnt
+          mount "/dev/disk/by-label/boot" /boot
```

Comment:
We can do this because of the label was prepared in create-datastore section, right? And there is no race since we run capture-macs After=create-datastore.service. Correct?

Reply:
We can do this because of the label was prepared in create-datastore section, right?
- yes, the PXE send's the kargs for the create DS and this labels the DS volume as boot 
And there is no race since we run capture-macs After=create-datastore.service. Correct?
- yes, the DS script will first create DS with label boot and then we will search for this label and mount it and allow us to save the file